p2p/discover/table_test.go

```diff
@@ -382,7 +385,7 @@ func TestTable_revalidateSyncRecord(t *testing.T) {
 	r.Set(enr.IP(net.IP{127, 0, 0, 1}))
 	id := enode.ID{1}
 	n1 := wrapNode(enode.SignNull(&r, id))
-	tab.addSeenNode(n1)
+	tab.addSeenNodeSync(n1)
```

Comment:
Ideally other tests should not be changed when testing new functionality. Is there a reason it was necessary to modify these tests?

Reply:
Are you still planning to switch back to `addSeenNode` ?